hathor/cli/run_node.py

```diff
@@ -164,24 +171,34 @@ def create_wallet():
                 raise ValueError('Invalid type for wallet')
 
         tx_storage: TransactionStorage
-        if args.data:
-            if args.rocksdb_storage:
-                tx_storage = TransactionRocksDBStorage(path=args.data, with_index=(not args.cache))
-            else:
-                tx_storage = TransactionCompactStorage(path=args.data, with_index=(not args.cache))
-            self.log.info('with storage', storage_class=type(tx_storage).__name__, path=args.data)
-            if args.cache:
-                tx_storage = TransactionCacheStorage(tx_storage, reactor)
-                if args.cache_size:
-                    tx_storage.capacity = args.cache_size
-                if args.cache_interval:
-                    tx_storage.interval = args.cache_interval
-                self.log.info('with cache', capacity=tx_storage.capacity, interval=tx_storage.interval)
-                tx_storage.start()
-        else:
+        if args.memory_storage:
+            assert not args.data, '--data should not be used with --memory-storage'
```

Comment:
Change asserts to `raise ValueError()` or create a method `check_and_exit(not args.data, '--data should not be used with --memory-storage')` which will prints the message and exit the process.

Reply:
I created an util function `check_or_exit`, what do you think?

Reply:
Great! Thanks!